Examples/Tests/Langmuir/langmuir2d_analysis.py

```diff
@@ -40,7 +40,7 @@
            %np.max( np.abs( ( Ex[:32,:,0] - E_predicted ) / E_predicted ) ) )
 assert np.allclose( Ex[:32,:,0], E_predicted, rtol=0.1 )
 print( "absolute error: np.max( np.abs( Ex[32:,:,0] ) ) = %s" %np.max( np.abs( Ex[:32,:,0] ) ) )
-assert np.allclose( Ex[32:,:,0], 0, atol=1.e-4 )
+assert np.allclose( Ex[32:,:,0], 0, atol=1.e-2 )
```

Comment:
let's see how much precision we already get...

Reply:
@dpgrote not sure if this test fails do to a precision issue or due to a floating point representation issue. Do you maybe have an idea?

Reply:
```
absolute error: np.max( np.abs( Ex[32:,:,0] ) ) = 2768442368.0
```

Reply:
Now at
```
absolute error: np.max( np.abs( Ex[32:,:,0] ) ) = 175820.53125
```

Reply:
I checked the values and observe the following. For the relatively high density of the test case, typical E-fields are in the order of `1e9 V/m`. Regarding machine precision noise, "zero" would be in the following orders:
* double (15 significant digits): 1.e-6 (which we see, "a bit" better even)
* single (6 significant digits); 1.e3 (roughly what we see here, "a bit" worse even)

note "a bit" is about two orders of magnitude, just eye-balling ;)